forge/src/main/java/com/wynntils/forge/WynntilsModForge.java

```diff
@@ -23,6 +24,9 @@ public WynntilsModForge() {
 
         File modFile = new File(path.toUri());
 
+        // Enable stencil support
+        Minecraft.getInstance().getMainRenderTarget().enableStencil();
```

Comment:
So in Forge there's a simple call for this?! :-o What does that do? The same as what you do "manually" in Fabric?

Reply:
Yeah, basically